docs/sidebars.js

```diff
@@ -40,6 +40,7 @@ const sidebars = {
         'basic/options',
         'basic/authenticating-oauth',
         'basic/socket-mode',
+        'basic/app-assistant'
```

Comment:
lolll i had it in advanced but Kaz thought it was better in basic

This brings up the bigger issue that categorizing docs by _basic_ and _advanced_ is not a very effective way of divying up docs. It's arbitrary, _and_ can be be belittiling ("wow i feel dumb that this 'basic' thing isn't working for me"). basic/simple/easy are all banned words for the docs team, at least. 

I think solution-focused categories would make more sense. A dev would think "i want to add interactivity" and would then go to an interactivity category. I don't devs are thinking "i want to do something advanced"

I haven't drafted a change because it felt like i'd be stepping on yall's toes, but something to think about

Reply:
I feel like we should align with such principles that your team has converged on - makes sense to me at least! But I defer to Kaz. Can shelve this topic for now and revisit in the future.

Reply:
Great discussion! Here are the reasons I believe it's appropriate to include it in the "basic" section:
- We have app.event listener docs in the "basic" section, and this assistant is essentially a collection of app.event listeners with a few extras like context store. Therefore, I don't think it's more advanced compared to the existing docs.
- Labeling it as "advanced" might discourage developers from trying it out, even though it's not difficult to implement. If we categorize this as "advanced," it could be a barrier for new feature adoption.

Aside from this specific decision, I've been considering the need to revisit the "basic" vs "advanced" structure. Back in 2019, we published bolt-js documents in this format. It was reasonable then because bolt-js had fewer features, and many developers weren't yet familiar with this framework. Now that bolt-js/python/java have many users and their concepts are understood by Slack app developers, I don't think removing the top categories (basic and advanced) would cause much confusion. We don't need to decide in this PR, but it's worth considering. Let's continue discussing this in Slack channels!